public/app/features/alerting/unified/components/rule-editor/CloudEvaluationBehavior.tsx

```diff
@@ -27,7 +27,7 @@ export const CloudEvaluationBehavior = () => {
     <RuleEditorSection stepNo={3} title="Set evaluation behavior">
       <Field
         label="Pending period"
-        description='Period the threshold condition must be met to trigger the alert. Selecting "None" triggers the alert immediately once the condition is met.'
+        description='The period in which the threshold condition must be met. Selecting "None" triggers the alert immediately once the condition is met.'
```

Comment:
how about Period during which the threshold condition must be met to trigger an alert.

Reply:
Let's go with that :)